app/views/appeal/discussion.scala

```diff
@@ -158,7 +158,7 @@ object discussion:
       form3.globalError(form),
       form3.group(
         form("text"),
-        if (isNew) "Create an appeal" else "Add something to the appeal",
+        if (isNew) "Create an appeal" else "Add something to your case",
```

Comment:
What's the bad grammar here? Also this phrasing has been there since the beginning, not sure if it's worth changing over this.